wiki/Contests/SVS/1/en.md

```diff
@@ -0,0 +1,91 @@
+---
+tags:
+  - svs
+  - svs1
+  - svs 1
+  - svs#1
+---
+
+# Slider Velocity Showdown
+
+![SVS logo](/wiki/contests/svs/img/Banner.png)
+
+The **Slider Velocity Showdown** (***SVS***) was the 1st iteration of the osu!taiko mapping contest with a focus on creativity and innovation. This contest was open to all that want to flex their creative muscles, whether they’ve never placed an inherited point in your life or if they’ve been mapping for years.
+
+## Contest schedule
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| Registration phase | 2021-07-01/2021-08-01 23:59 |
+| Submission phase | 2021-08-01/2021-09-01 23:59 |
+| Judging phase | 2021-09-01/2021-09-14 |
+| Results announcement | 2021-09-18 |
```

Comment:
the phases overlap according to the timestamp, is that intentional?

Reply:
adjusted